web/src/Backend.js

```diff
@@ -1,129 +1,175 @@
 import * as Setting from "./Setting";
 
 export function getAdapters() {
-    return fetch(`${Setting.ServerUrl}/api/get-adapters`, {
-        method: "GET",
-        credentials: "include"
-    }).then(res => res.json());
+  return fetch(`${Setting.ServerUrl}/api/get-adapters`, {
+    method: "GET",
+    credentials: "include",
+  })
+    .then(res => res.json())
+    .catch(err => console.error(err));
 }
 
 export function getAdapter(adapterId) {
-    return fetch(`${Setting.ServerUrl}/api/get-adapter?id=${adapterId}`, {
-        method: "GET",
-        credentials: "include"
-    }).then(res => res.json());
+  return fetch(`${Setting.ServerUrl}/api/get-adapter?id=${adapterId}`, {
+    method: "GET",
+    credentials: "include",
+  })
+    .then(res => res.json())
+    .catch(err => console.error(err));
 }
 
 export function updateAdapters(adapters) {
-    return fetch(`${Setting.ServerUrl}/api/update-adapters`, {
-        method: 'POST',
-        credentials: 'include',
-        body: JSON.stringify(adapters),
-    }).then(res => res.json());
+  return fetch(`${Setting.ServerUrl}/api/update-adapters`, {
+    method: "POST",
+    credentials: "include",
+    body: JSON.stringify(adapters),
+  })
+    .then(res => res.json())
+    .catch(err => console.error(err));
 }
 
 export function updateAdapter(adapter) {
-    return fetch(`${Setting.ServerUrl}/api/update-adapter`, {
-        method: 'POST',
-        credentials: 'include',
-        body: JSON.stringify(adapter),
-    }).then(res => res.json());
+  return fetch(`${Setting.ServerUrl}/api/update-adapter`, {
+    method: "POST",
+    credentials: "include",
+    body: JSON.stringify(adapter),
+  })
+    .then(res => res.json())
+    .catch(err => console.error(err));

```

Comment:
Thanks @anishagg17 for working on this. But I don't think print down the error is the right way to catch it. You need to convey a friendly message to the user also about the error because silently catching is not a good for the user.

Reply:
will do 👍